app/components/agenda/move.hbs

```diff
@@ -0,0 +1,38 @@
+<label class="form__label">Verplaats agendapunt
+  <div class="grid">
+    <div class="col--6-12">
+      {{#power-select
+           class="select select--block u-spacer--tiny"
+           renderInPlace=true
+           placeholder="Selecteer positie[@TN]"
+           options=this.locationOptions
+           verticalPosition="above"
+           searchEnabled=false
+           selected=@selectedLocation
+           allowClear=true
+           onchange=@selectLocation
+           as |location|}}
+        {{location.name}}
+      {{/power-select}}
+    </div>
+    {{#if @showAfterAgendapuntOptions}}
+      <div class="col--6-12">
+        {{#power-select
```

Comment:
here we could use power select, because we offer type ahead. but it should definitely be converted to octane syntax.